Userland/Libraries/LibWeb/Layout/ImageBox.cpp

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include "Build/i686/Userland/Libraries/LibWeb/CSS/ValueID.h"
+#include "LibWeb/CSS/StyleValue.h"
```

Comment:
Including a generated file directly will make this only work on i686. :^) If you include it like this, the compiler will find whichever one is correct for the current architecture.

```suggestion
#include <LibWeb/CSS/ValueID.h>
#include <LibWeb/CSS/StyleValue.h>
```

(And then sort the includes, of course.)

Reply:
These includes are automatically added by clangd for some reason. I will remove them anyway :^)